ieeeuoft/hackathon-template
# Acknowledgement Form Connected to Formik

## Overview

- Added `/acknowledgement` path
- Created acknowledgement page and form
- Create a modal for "Waiver of Liability and Hold Harmless Agreement" 

Page:
<img width="1440" alt="Screen Shot 2021-05-06 at 11 41 57 PM" src="https://user-images.githubusercontent.com/42653157/117394760-b6e1b380-aec4-11eb-8c4d-6fde849d1d83.png">

Modal:
<img width="1440" alt="Screen Shot 2021-05-06 at 11 42 03 PM" src="https://user-images.githubusercontent.com/42653157/117394765-b812e080-aec4-11eb-819b-9a9f8e7efa98.png">

## Unit Tests Created

- Added unit tests for AcknowledgementForm


## Steps to QA

- Go to `localhost:3000/acknowledgement`
- Fill the form out and press submit, you will see an alert of the values you inputted
- The form button should be disabled if everything is not filled
- Click the link "Waiver of Liability and Hold Harmless Agreement" to confirm the modal pops up
